server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/PartitionPlanController.java

```diff
@@ -41,24 +39,21 @@ public class PartitionPlanController {
 
     @Autowired
     private PartitionPlanService partitionPlanService;
-    @Autowired
-    private PartitionPlanServiceV2 partitionPlanServiceV2;
 
     @RequestMapping(value = "/partitionPlans", method = RequestMethod.GET)
-    public SuccessResponse<DatabasePartitionPlan> getPartitionPlans(@RequestParam Long databaseId,
+    public SuccessResponse<String> getPartitionPlans(@RequestParam Long databaseId,
             @RequestParam(required = false) Long flowInstanceId) {
-        return Responses
-                .success(partitionPlanService.findRangeTablePlan(databaseId, flowInstanceId));
+        throw new UnsupportedOperationException("UnSupported now");
     }
 
     @RequestMapping(value = "/partitionPlans/exists", method = RequestMethod.GET)
     public SuccessResponse<Boolean> exist(@RequestParam("databaseId") Long databaseId) {
-        return Responses.success(partitionPlanService.hasConnectionPartitionPlan(databaseId));
+        throw new UnsupportedOperationException("UnSupported now");
```

Comment:
Are multiple partition plans now allowed for the same database?

Reply:
no, a database can only be attached to an enabled partition plan task.

there is an api `candidatePartitionPlanTables` which return list of `PartitionPlanDBTable` of a database. `PartitionPlanDBTable` will tell the front end that which table has been configured and which is not~